app/routes/formbuilder/new.js

```diff
@@ -10,15 +10,11 @@ export default class FormbuilderNewRoute extends Route {
     const newForm = await this.store.createRecord('generated-form', {
```

Comment:
This code runs on route load. This means that it creates a new empty form before the user specified any title or description